new-website/src/3.x/docs/cli-commands/repack-init.md

```diff
@@ -0,0 +1,43 @@
+# repack-init
+
+`@callstack/repack-init` is a command-line tool that helps you to set up Re.Pack in your React Native project. It installs all required dependencies and configures your project to use Re.Pack.
+
+## Usage
+
+```bash
+npx @callstack/repack-init [options]
+```
+
+## Options
+
+### `-c`, `-custom-version`
```

Comment:
let's also change the order here so that the long version is first and the short alias is second + let's remove the code markers here so that it's a normal text